src/containers/Review/ReviewSubmit.tsx

```diff
@@ -21,12 +25,29 @@ type ReviewSubmitProps = {
 
 const ReviewSubmit: React.FC<ReviewSubmitProps> = (props) => {
   const {
-    structure: { thisReview, assignment, canApplicantMakeChanges },
+    structure: { info, thisReview, assignment, canApplicantMakeChanges },
   } = props
+
+  const [fullyAssigned, setFullyAssigned] = useState(false)
```

Comment:
This is that same "derived state" issue I mentioned on something else a while back -- you don't need to create a React state variable for values that are derived purely from existing state variables. 

In this case `fullyAssigned` is just `assignedQuestions?.totalCount === reviewableQuestions?.totalCount`, so you there's no can just state the value directly and no need to wrap it in a `useEffect` either.